build.sbt

```diff
@@ -15,8 +15,8 @@ import java.util.Calendar
 import scala.language.postfixOps
 
 name := "customs-declarations"
-scalaVersion := "2.12.13"
-targetJvm := "jvm-1.8"
+scalaVersion := "2.13.10"
```

Comment:
We can now bump this to 2.13.11: https://github.com/scala/scala/releases/tag/v2.13.11